python/tests/test_drawing.py

```diff
@@ -1750,6 +1750,15 @@ def test_draw_simple_ts(self):
         svg = ts.draw_svg()
         self.verify_basic_svg(svg, width=200 * ts.num_trees)
 
+    def test_draw_integer_breaks_ts(self):
+        r_map = msprime.RecombinationMap.uniform_map(1000, 0.001, discrete=True)
```

Comment:
``discrete=True`` is msprime 1.0 and hasn't been released. Need to use the ``num_loci=1000`` argument instead.

Reply:
Yep, sorry. Fixed.